crates/ruff_python_resolver/src/search.rs

```diff
@@ -0,0 +1,282 @@
+//! Determine the appropriate search paths for the Python environment.
+
+use std::collections::HashMap;
+use std::ffi::OsStr;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+use log::debug;
+
+use crate::config::Config;
+use crate::module_descriptor::ImportModuleDescriptor;
+use crate::python_version::PythonVersion;
+use crate::{host, SITE_PACKAGES};
+
+/// Find the `site-packages` directory for the specified Python version.
+fn find_site_packages_path(
+    lib_path: &Path,
+    python_version: Option<PythonVersion>,
+) -> Option<PathBuf> {
+    if lib_path.is_dir() {
+        debug!(
+            "Found path `{}`; looking for site-packages",
+            lib_path.display()
+        );
+    } else {
+        debug!("Did not find `{}`", lib_path.display());
+    }
+
+    let site_packages_path = lib_path.join(SITE_PACKAGES);
```

Comment:
shouldn't `python{major}.{minor}` (unix) or `Python` (iirc windows) we in between the two? i've never seen the version without it

Reply:
Can you check the structure on Windows for me?

Reply:
sorry windows actually uses `Lib/site-packages`

![image](https://github.com/astral-sh/ruff/assets/6826232/eba30280-1aa2-40b1-a3bb-17dba7b9b75b)